matter_server/client/client.py

```diff
@@ -4,23 +4,20 @@
 
 import asyncio
 import logging
-from types import TracebackType
-from typing import TYPE_CHECKING, Any, Callable, Dict, Final, Optional, cast
 import uuid
+from typing import TYPE_CHECKING, Any, Final, cast
```

Comment:
I prefer having the imports sorted :abcd: regardless if they are imported with `import ...` or with `from ... import`. Can we adjust that rule? We have this sorted like that in core.

Reply:
@frenck do you know how ?